app-frontend/src/app/components/common/search/search.js

```diff
@@ -72,7 +79,7 @@ class SearchController {
 
     clearSearch() {
         this.searchText = '';
-        this.onSearch();
+        this.onSearch({value: null});
```

Comment:
This seems to cause the clear search feature fail in other places where this search component is also used.

Reply:
I'm going to update the other locations that we have a paginated search list to use the pagination service and updated search component api